### PR TITLE
trial balloon for tranfer naming

### PR DESCRIPTION
**Pull request by fosterlynn**
Please do not merge, this is to be able to discuss.  Examples will be upgraded after we decide on naming and confirm the pattern.

### CODE REVIEW

File: docs/introduction/flows.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ All flows use an action property to designate what the flow is doing and how it
 * vf:pass - possible output of a testing or reviewing process, indicating the resource passed, the same resource will appear in input with *vf:accept* verb
 * vf:fail - possible output of a testing or reviewing process, indicating the resource failed, the same resource will appear in input with *vf:accept* verb
 * vf:service - new service produced and delivered (a service implies that an agent actively receives the service)
-* vf:transfer - give rights and/or possession of a resource from one agent to another
+* vf:transfer-accounting - give full (in the human realm) rights and responsibilities to another agent, without transferring physical custody
```
Comment (bhaugen) [inline docs/introduction/flows.md]: Will people understand transfer-accounting? I understand not wanting to say "transfer-ownership" but maybe "transfer-rights"? Or "transfer-all-rights"? 
(But I like the description.)

File: docs/introduction/flows.md [hunk 2]
```diff
@@ -59,7 +59,9 @@ All flows use an action property to designate what the flow is doing and how it
 * vf:pass - possible output of a testing or reviewing process, indicating the resource passed, the same resource will appear in input with *vf:accept* verb
 * vf:fail - possible output of a testing or reviewing process, indicating the resource failed, the same resource will appear in input with *vf:accept* verb
 * vf:service - new service produced and delivered (a service implies that an agent actively receives the service)
-* vf:transfer - give rights and/or possession of a resource from one agent to another
```
Comment (bhaugen) [inline docs/introduction/flows.md]: I like the change to different named types of transfers.

File: docs/introduction/flows.md (hunk 1 above)
Comment (bhaugen) [reply docs/introduction/flows.md]: But I could live happily with transfer-accounting.

**Comment by fosterlynn** [comment]
Used all-rights as suggested.  Clarified text sections, fixed examples.

Also changed load/unload to pickup/dropoff, to include transportation of people.  And a few misc. corrections.

**Pull request merged**